frontend/ui/widget/frontlightwidget.lua

```diff
@@ -25,7 +25,7 @@ local FrontLightWidget = InputContainer:new{
     title_face = Font:getFace("tfont", 22),
     width = nil,
     height = nil,
-    fl_cur = 0,
+    fl_cur = 1,
     fl_min = 0,
```

Comment:
Not related to your fix. But all of these attributes can be initialized once here through powerd so you don't have to do every time `init` is called.